core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -5006,6 +5006,17 @@ secret_configs:
   - cluster: app.ci
     name: ci-image-mirror
     namespace: ci
+- from:
+    config.json:
+      dockerconfigJSON:
+      - auth_field: sa.multi-arch-builder-controller.multi01.token.txt
```

Comment:
@hongkailiu If I understand correctly, this will be generated by https://github.com/openshift/release/pull/42551/files#diff-a1afb2e1637a7b7fd823c19e62f444a3cf739420a7b4a5d2483bfd089977f9efR246-R249, right?